cmake/modules/kconfig.cmake

```diff
@@ -175,7 +178,7 @@ endforeach()
 unset(EXTRA_KCONFIG_OPTIONS)
 get_cmake_property(cache_variable_names CACHE_VARIABLES)
 foreach (name ${cache_variable_names})
-  if("${name}" MATCHES "^CONFIG_")
+  if("${name}" MATCHES "^${KCONFIG_NAMESPACE}_")
     # When a cache variable starts with 'CONFIG_', it is assumed to be
```

Comment:
Several comments in this file have become stale now -- they still reference `CONFIG`, but this is now `${KCONFIG_NAMESPACE}`. Intentional?

Reply:
no, thanks for spotting.

Should be fixed now (unless I missed some locations).